camera_matrix:
  cols: 3
  data: [673.2235815227677, 0.0, 350.61542491784013, 0.0, 673.9155164096895, 253.6555424620399,
    0.0, 0.0, 1.0]
  rows: 3
distortion_coefficients:
  cols: 5
  data: [0.07335667221962282, -0.9053116565379455, 0.003965169729883846, 0.0035118811273455986,
    2.1401816560333384]
  rows: 1
image_height: 480
image_width: 640
